api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java

```diff
@@ -146,8 +142,6 @@
   private static final String LOGGED_IN_USER_EMAIL = "bob@gmail.com";
   private static final String OTHER_USER_EMAIL = "alice@gmail.com";
   private static final String BUCKET_NAME = "workspace-bucket";
-  private static final Instant NOW = Instant.now();
-  private static final FakeClock CLOCK = new FakeClock(NOW, ZoneId.systemDefault());
```

Comment:
We have ~20 other tests which instantiate their own FakeClock.  We should consider removing those as well.